src/modules/auth/component/SignUp/SignUp.tsx

```diff
@@ -74,54 +78,81 @@ export function SignUp() {
         <Grid container spacing={2}>
           <Grid item sm={12}>
 
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit(onSubmit)}>
               <Grid container spacing={6}>
                 <Grid item xs={12}>
-                  <TextField
-                    label="Email"
-                    placeholder="email@gmail.com"
-                    type="email"
+                  <Controller
```

Comment:
Create EmailField component that uses TextField, that should have been created from one of the previous comments. It should just use this TextField and pass rules, control, type and placeholder as the props